.github/workflows/ci.yml

```
@@ -8,57 +8,57 @@ on:
 
 jobs:
   chore:
-    name: 'Testing chores'
+    name: "Testing chores"
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - run: |
           git fetch --no-tags --unshallow origin HEAD main
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: "v14.19.2"
 
-      - name: 'Check that the cache files are consistent with their remote sources'
+      - name: "Check that the cache files are consistent with their remote sources"
         run: |
           if [[ $(git diff --name-only "$(git merge-base origin/"$TARGET_BRANCH" HEAD)" HEAD -- .yarn/cache | wc -l) -gt 0 ]]; then
-            yarn --immutable --immutable-cache --check-cache
+            corepack yarn --immutable --immutable-cache --check-cache
           fi
         shell: bash
         if: |
           github.event.pull_request != ''
         env:
           TARGET_BRANCH: ${{github.event.pull_request.base.ref}}
 
-      - name: 'Check for type errors'
-        run: yarn typecheck
+      - name: "Check for type errors"
+        run: corepack yarn typecheck
 
   build:
     strategy:
       fail-fast: false
       matrix:
         node:
-        - 14
+          - "v14.19.2"
         platform:
-        - ubuntu-latest
-        - macos-latest
-        - windows-latest
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
 
-    name: '${{matrix.platform}} w/ Node.js ${{matrix.node}}.x'
+    name: "${{matrix.platform}} w/ Node.js 14.x"
```

Imo we should keep the `matrix.node` variable here (just remove the `.x`)
```suggestion
    name: "${{matrix.platform}} w/ Node.js ${{matrix.node}}"
```

---

Ah I see - makes sense then, combined with your other comment 👍